fractal-puppy/Engine
# Changed the configuration of the mouse clicks and enhanced its interaction with the enemies

To test:
- Check that now left click makes the player move
- Check that pressing shift the player starts doing the basic attack
- Check that the animation transition from attacking with the shift to moving works
- Check that when clicking on an enemy, the player goes to the enemy to attack it
- Check that when clicking on an enemy being at melee distance the player attacks it
You can use the scene EnemiesTest2 to try these features.
